tools/msmetaenhancer/msmetaenhancer.xml

```diff
@@ -8,6 +8,7 @@
 
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">msmetaenhancer</requirement>
+        <requirement type="package" version="0.14.0">matchms</requirement>
```

Comment:
This is not necessary

Reply:
You are right, its installed with `MSME`.